source/conf.py

```diff
@@ -105,7 +105,8 @@
     'sphinxcontrib.asciinema',
     'sphinx_toolbox.confval',
     'sphinx-prompt',
-    'myst_parser'
+    'myst_parser',
+    'sphinx_reredirects'
```

Comment:
You can put a comma here:
```suggestion
    'sphinx_reredirects',
```
And future diffs will be cleaner because it doesn't change the previous line.

Reply:
I agree! I've noticed that this hasn't been the case in the documentation, so I was just following precedent, but there is no good reason to not do it properly.